src/components/icon/update.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import IconContainer from './icon-container';
+
+const Update = ({ color = '#ffd23f' }) => (
+  <IconContainer>
+    <>
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M8.87147 16.1285C5.00548 16.1285 1.87147 12.9945 1.87147 9.12852C1.87148 5.39457 4.79506 2.34346 8.47801 2.1394L9.07348 3.13186C9.00642 3.12964 8.93908 3.12853 8.87148 3.12853C5.55777 3.12852 2.87148 5.81482 2.87147 9.12852C2.87147 12.4422 5.55777 15.1285 8.87147 15.1285C12.1852 15.1285 14.8715 12.4422 14.8715 9.12853C14.8715 6.58053 13.2832 4.40349 11.0427 3.53343L12.1279 2.93052C14.3538 4.10242 15.8715 6.43824 15.8715 9.12853C15.8715 12.9945 12.7375 16.1285 8.87147 16.1285Z"
+        fill={color}
+      />
+      <path
+        d="M15.7636 3.22485L11.2678 2.89093L12.3647 7.2637"
+        stroke={color}
+      />
+    </>
+  </IconContainer>
+);
+
+export default Update;
```

Comment:
Is this file still being used?

Reply:
No- i didn't delete the icon though, and the color is a prop. We can cut it out during code review if it is still not needed.